lib/server/routes/frames.js

```diff
@@ -351,11 +351,35 @@ FramesRouter.prototype.getFrameById = function(req, res, next) {
  */
 FramesRouter.prototype._definitions = function() {
   return [
-    ['POST', '/frames', this._limiter(defaults), this._verify, this.createFrame],
-    ['PUT', '/frames/:frame', this._limiter(defaults), this._verify, this.addShardToFrame],
-    ['DELETE', '/frames/:frame', this._limiter(defaults), this._verify, this.destroyFrameById],
-    ['GET', '/frames', this._limiter(defaults), this._verify, this.getFrames],
-    ['GET', '/frames/:frame', this._limiter(defaults), this._verify, this.getFrameById]
+    ['POST',
```

Comment:
In regards to the linting here, we've made an exception here for readability for other definitions: https://github.com/Storj/bridge/blob/master/lib/server/routes/buckets.js#L1418